### PR TITLE
Add style for main navigation and button

### PR DESCRIPTION
**Pull request by kylefiedler**
screenshot 2015-09-18 15 59 58

### CODE REVIEW

**Comment by smharley** [comment]
looks dope!

File: app/assets/stylesheets/components/_hero.scss [hunk 1]
```diff
@@ -6,6 +6,13 @@
   .button {
     margin-top: 3rem;
   }
+
+  .github-icon {
+    @include size(1rem);
+    fill: $base-font-color;
+    margin-right: 1rem;
+    @include position(relative, 2px null null null);
```
Comment (kylefiedler) [inline app/assets/stylesheets/components/_hero.scss]: :abc:

**Comment by smharley** [comment]
code looks good, too! :ship:

File: app/assets/stylesheets/layout/_header.scss [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
   a {
     display: block;
+    color: rgba($base-font-color, 0.5);
     background-color: darken($base-background-color, 10%);
     padding: $small-spacing $base-spacing;
```
Comment (kylefiedler) [inline app/assets/stylesheets/layout/_header.scss]: :abc:

**Comment by JuanitoFatas** [comment]
:shipit:

**Pull request merged**